### DISCUSSION TITLE
Making Mantine Table Cells Editable

### DISCUSSION THREAD

**Discussion by Nathan-Bernardo** [mantinedev/mantine#4056] [Q&A] [answered]
Hi guys, I've used a few libraries such as React-Table and Ant Design, and both have features for editing table cells.  I'm wondering if it is possible to make Mantine cells editable as well? If so, what would be your approach?

**Comment by rtivital** [comment]
Mantine Table component provides styles, there is no extra functionality built into it. You can render input inside cells to make them editable.

**Comment by ForeshadowRU** [comment, marked as answer]
Or you can try community-built data-grids.

https://github.com/icflorescu/mantine-datatable
https://www.mantine-react-table.com/
